requirements.txt

```diff
@@ -5,3 +5,5 @@ requests==2.18.4
 requests_mock==1.4.0
 tldextract==2.2.0
 yattag==1.10.0
+mysqlclient==1.3.13
```

Comment:
Not sure if this is a pip install-able module yet. Didn't work for me personally.

Reply:
https://stackoverflow.com/questions/35991403/pip-install-unroll-python-setup-py-egg-info-failed-with-error-code-1

Reply:
I followed these steps and it didn't solve the issue.

Reply:
Did you try other solutions?

Reply:
I'll try the rest in a little and let you know if they work.

Reply:
Ok

Reply:
Can you post the error message?

Reply:
Failing for me as well
here's the error code
`root@kali:~/github/TorBot# pip install mysqlclient
Collecting mysqlclient
  Using cached https://files.pythonhosted.org/packages/ec/fd/83329b9d3e14f7344d1cb31f128e6dbba70c5975c9e57896815dbb1988ad/mysqlclient-1.3.13.tar.gz
    Complete output from command python setup.py egg_info:
    /bin/sh: 1: mysql_config: not found
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-41snh5qi/mysqlclient/setup.py", line 18, in <module>
        metadata, options = get_config()
      File "/tmp/pip-install-41snh5qi/mysqlclient/setup_posix.py", line 53, in get_config
        libs = mysql_config("libs_r")
      File "/tmp/pip-install-41snh5qi/mysqlclient/setup_posix.py", line 28, in mysql_config
        raise EnvironmentError("%s not found" % (mysql_config.path,))
    OSError: mysql_config not found
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-41snh5qi/mysqlclient/
`

Reply:
@shivankar-madaan Try this ` pip install --upgrade setuptools`

Reply:
yes tried all the steps in the stackoverflow link

Reply:
See this also
https://stackoverflow.com/questions/25865270/how-to-install-python-mysqldb-module-using-pip